### ISSUE TITLE
lintDirtyModulesOnly doesn't works on Windows

### ISSUE THREAD

**Issue by vidal7** [webpack-contrib/stylelint-webpack-plugin#185]
- Operating System: Windows 10
- Node Version: 10.15.3
- NPM Version: 6.4.1
- webpack Version: 4.39.3
- stylelint-webpack-plugin Version: 1.0.1

### Expected Behavior

lintDirtyModulesOnly options set to true must lint only dirty modules

### Actual Behavior

Is linting no modules at all. Method getChangedFiles of LintDirtyModulesPlugin is always returning an empty array.

### Code

```js
new StyleLintPlugin({
            emitError: true,
            lintDirtyModulesOnly: true
        }),
```

### How Do We Reproduce?

Just be on windows. It seems that glob pattern is not working with back-slashes on windows. It needs to be escaped to works.

For example, when changing `C:\src\communs\containers\debug\debug.scss` file, LintDirtyModulesPlugin pickup this file in changed files but failed to be push to the changedFiles array because of the micromatch glob pattern. The glob pattern needs to be `C:\\src\\**\\*.s?(c|a)ss` instead of `C:\src\**\*.s?(c|a)ss`
